public/css/style.css

```diff
@@ -18,11 +27,74 @@ main {
     padding: 5px 10px;
 }
 
+h1 {
+    color: white;
+}
+
 a {
     text-decoration: none;
 }
 
+nav a {
+    color: white;
+    padding-left: 30px;
+    padding-right: 30px;
+}
+
+nav a:hover {
+    background-color: black;
+    padding: 27.5px;
+    color: white;
+}
+
 .profile-small-section {
     height: 170px;
     overflow: auto;
 }
+
+.grid-container {
+    display: grid;
+    grid-template-areas: 
+        'header-left header-left header-left header-right header-right header-right'
+        'left-one left-one center-one center-one right-one right-one';
+    grid-gap: 10px;
+    padding: 10px;
+}
+
+.grid-item-1 {
+    grid-area: header-left;
+    font-size: 30px;
+    text-align: center;
+}
+
+.grid-item-2 {
+    grid-area: header-right;
+    font-size: smaller;
+    text-align: justify;
+}
+
+.grid-item-3 {
+    grid-area: left-one;
+    font-size: smaller;
+    text-align: justify;
+}
+
+.grid-item-4 {
+    grid-area: center-one;
+    font-size: smaller;
+    text-align: justify;
+}
+
+.grid-item-5 {
+    grid-area: right-one;
+    font-size: 30px;
+    text-align: center;
+}
+
+.card-body {
```

Comment:
"card-body" is predefined class in bootstrap. Not ideal to customize unless you want all card-body in this app to be with this styling

Reply:
I did want all the cards to be styled the same